src/Components/Dropdown/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Dropdown } from "react-bootstrap";
+
+export const DropDown: React.FC = () => {
+  return (
+    <div>
+      <Dropdown>
+        <Dropdown.Toggle variant="success" id="dropdown-basic">
+          ISKCON
+        </Dropdown.Toggle>
+        <Dropdown.Menu>
+          <Dropdown.Item href="/Pages/dropdowninfo">History</Dropdown.Item>
+          <Dropdown.Item href="/Pages/Servicespage/IskconYouthForum">
+            Iskcon Youth Forum - IYF
+          </Dropdown.Item>
+          <Dropdown.Item href="/Pages/Servicespage/GuestHouse">
+            Prabhupada Ashrya Guest House
+          </Dropdown.Item>
+          <Dropdown.Item href="/Pages/Servicespage/Gaushala">
+            Goverdhan Gaushala
+          </Dropdown.Item>
+          <Dropdown.Item href="/Pages/Servicespage/DeityWorship">
+            Deity Worship
+          </Dropdown.Item>
+          <Dropdown.Item href="/Pages/Servicespage/DevoteeKitchen">
+            Devotee Kitchen
+          </Dropdown.Item>
+          <Dropdown.Item href="/Pages/Servicespage/KirtanAtIskcon">
+            Kirtan At Iskcon
+          </Dropdown.Item>
+          <Dropdown.Item href="/Pages/Servicespage/BookDistribution">
+            Book Distribution
+          </Dropdown.Item>
+        </Dropdown.Menu>
```

Comment:
I don't see routes set for all these paths in App.tsx. Why is it here? Also, I said the path should not start with `Pages` and should not have capital letters

Reply:
Also you can use map function here